hardhat.config.ts

```diff
@@ -154,38 +155,38 @@ const config: HardhatUserConfig = {
       // gasPrice: 2000000000,
     },
   },
-  etherscan: {
```

Comment:
@suNrisEinMyeYes Why etherscan configuration is commented? How to verify contracts?

Reply:
![image](https://user-images.githubusercontent.com/40171291/193070560-8071ea05-4b43-4322-9971-cc51c4f47ded.png)

Reply:
I didn't implement this part, so I don't know why it is marked as error

Reply:
Works fine on my side. Did you try to install latest dependencies `yarn install`?